ui_framework/components/table/table_row.test.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { render } from 'enzyme';
+import { requiredProps } from '../../test/required_props';
+
+import {
+  KuiTableRow,
+} from './table_row';
+
+import {
+  KuiTableRowCell
+} from './table_row_cell';
+
+test('renders KuiTableRow', () => {
+  const component = <KuiTableRow { ...requiredProps }><KuiTableRowCell>hi</KuiTableRowCell></KuiTableRow>;
```

Comment:
Same with this, we should have style rules for this type of thing.  Maybe something like, `don't put nested tags on the same line`.